packages/landing/static/index.css

```diff
@@ -196,4 +196,19 @@ button {
   left: 48px;
   width: 600px;
   height: 40px;
+}
+
+.date-range {
+  background:rgba(255, 255, 255, 0.8);
+  box-shadow: 0 0 0.5rem 0.5rem rgb(0 0 0 / 10%);
+  border: 2px solid rgba(255, 255, 255);
+  border-radius: 0.5rem;
+  padding: 0 1rem 1rem 1rem;
```

Comment:
I dont think we use any other `rem` for measurements, might just be best to use what everything else is using 4/8px increment.s